### PR TITLE
QLB Yearly + adam removal

### PR DESCRIPTION
**Pull request by blister**
Adds the `.qlb yearly` option to display a queue leaderboard for the previous 365 days.

Also removes references to asking adam for help if anything is wrong and directs users to report bugs to the RSC Development Committee.

### CODE REVIEW

File: sixMans/sixMans.py [hunk 1]
```diff
@@ -868,6 +888,27 @@ async def monthly(self, ctx: Context, player: discord.Member = None, *, queue_na
         player = player if player else ctx.author
         await ctx.send(embed=self.embed_rank(player, sorted_players, queue_name, queue_max_size, "Monthly"))
 
+    @commands.guild_only()
+    @rank.command(aliases=["year", "yr"])
+    async def yearly(self, ctx: Context, player: discord.Member = None, *, queue_name: str = None):
```
Comment (BenjaminHunt) [inline sixMans/sixMans.py:893]: A little weird how qlb and rank swap grammar patterns between "yearly" and "year", "monthly" and "month", but its not a big deal. Maybe a small change we can make in the future.

File: sixMans/game.py [hunk 2]
```diff
@@ -585,7 +585,7 @@ async def post_more_lobby_info(self, helper_role=None, invalid=False):
                 "the `winning_team` parameter is either `Blue` or `Orange`. Both teams will need to verify the results.\n\nIf you wish to cancel "
                 "the game and allow players to queue again you can use the `{0}cg` command. Both teams will need to verify that they wish to "
                 "cancel the game.".format(self.prefix), inline=False)
-        help_message = "If you think the bot isn't working correctly or have suggestions to improve it, please contact adammast."
+        help_message = "If you think the bot isn't working correctly or have suggestions to improve it, please contact the RSC Development Committee."
```
Comment (BenjaminHunt) [inline sixMans/game.py:588]: This probably is an improvement over adammast considering we'll be maintaining it now. I wonder if we can put out a more specific point of contact, or info on how to contact us. Probably good for now.

**Review by BenjaminHunt — APPROVED** [review]
There's a couple things we can look at updating in the future, but these changes look good.

👍

File: sixMans/sixMans.py (hunk 1 above)
Comment (blister) [reply sixMans/sixMans.py:893]: Yeah, I had never actually used the rank command before so wasn't certain why it followed a different flow. Didn't want to rock the boat too much and force that change in this PR, but happy to normalize these in a different ticket. (Are we, as a team, using tickets? Or just trello?)

File: sixMans/game.py (hunk 2 above)
Comment (blister) [reply sixMans/game.py:588]: Yeah, at some point we'll probably want to expose some kind of ticketing system or something to the users.

**Pull request merged**